lib/formtastic-bootstrap/helpers/input_helper.rb

```diff
@@ -1,44 +0,0 @@
-module FormtasticBootstrap
-  module Helpers
-    module InputHelper
-      include Formtastic::Helpers::InputHelper
-
-      def input_class_with_const_defined(as)
-        input_class_name = custom_input_class_name(as)
-
-        if ::Object.const_defined?(input_class_name)
-          input_class_name.constantize
-        elsif FormtasticBootstrap::Inputs.const_defined?(input_class_name)
-          standard_input_class_name(as).constantize
-        elsif Formtastic::Inputs.const_defined?(input_class_name)
-          standard_formtastic_class_name(as).constantize
-        else
-          raise Formtastic::UnknownInputError, "Unable to find input class #{input_class_name}"
-        end
-      end
-      
-      def input_class_by_trying(as)
-        begin
-          custom_input_class_name(as).constantize
-        rescue NameError
-          begin
-            standard_input_class_name(as).constantize
-          rescue
-            standard_formtastic_class_name(as).constantize
-          end
-        end
-      rescue NameError
-        raise Formtastic::UnknownInputError, "Unable to find input class for #{as}"
-      end
```

Comment:
This was just added recently, and actually I don't think I released a gem with it. Is the InputHelper (singular) removed from Formtastic now? Should a similar method to input_class_by_trying go into the InputsHelper (plural) ?
https://github.com/mjbellantoni/formtastic-bootstrap/pull/112/files